corehq/apps/domain/templates/domain/admin/flags_and_privileges.html

```diff
@@ -99,23 +99,21 @@ <h1>Feature Flags</h1>
                                   data-bind="css: ('label-' + tagCssClass),
                                              text: tag"></span></td>
 
-                        <td>
-                            <a class="collapsed" data-toggle="collapse" aria-expanded="false"
-                               data-bind="visible: description,
-                                          attr: {href: '#toggleDescription-' + slug}">
-                                <i class="fa fa-angle-double-down"></i>
-                            </a>
-
+                        <td data-bind="click: showHideDescription">
                             <span data-bind="text: label"></span>
 
                             <span data-bind="visible: helpLink">
                                 (<a data-bind="attr: {href: helpLink}" target="_blank">docs</a>)
                             </span>
 
-                            <div class="collapse"
-                                 data-bind="visible: description,
-                                            attr: {id: 'toggleDescription-' + slug}">
-                                <div class="well" data-bind="text: description"></div>
+                            <span data-bind="visible: !!description">&#8230;</span>
```

Comment:
Is there a named entity you could use instead of the number?

Reply:
looks like there is, will change